src/Chirp.Web/Pages/Auth.cshtml.cs

```diff
@@ -15,14 +15,17 @@ public AuthModel(IAuthorRepository authorRepository)
 
     public ActionResult OnGet(bool? signOut)
     {
-        if (User.Identity.IsAuthenticated && signOut != null)
+        if (User?.Identity?.IsAuthenticated == true && signOut != null)
```

Comment:
Jeg kan godt forstå du gør det men du kunne overveje et eksplicit cast i stedet for implicit.

Reply:
if (User?.Identity != null && User.Identity.IsAuthenticated)

vil dette være bedre?

Reply:
Det synes jeg er fint!